Beautification Measures

CrazyGrape opened the pull request:
Added textures, .lang entries, and custom models.

Tweaked some code, primarily for tooltips and rendering purposes.

CrazyGrape committed:
Added Various Textures

-OreWand animated texture "Orecle"
-Generator textures
-Revised Ingot block textures
-GUI Textures (Generator and upgrade interface)
-Stone and Diamond Gears

CrazyGrape committed:
Added some .lang entries

CrazyGrape committed:
Modeled Frames

-Added oregenwand/Orecle tooltips in code and .lang
-Added tooltips for gears, fluids, frames, and the fluid tab
-Adjusted lead ore tooltip
-Added custom models for machine frames
     >.json models and blockstates
-Adjusted and fixed rendering for frame blocks
-Adjusted damage of combination wrench

SeanLatimer commented on src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java:
```diff
@@ -64,7 +64,7 @@
     private static IBlockState blockHarvest = null;
```
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "blockHarvest" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1068)

SeanLatimer commented:
SonarQube analysis reported 6 issues
* ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) 5 major
* ![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 1 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemWorldGenWand.java#L38](https://github.com/AlgorithmicsAnonymous/generitech/blob/7e1f33e8253730887b79a224cd65274b02577dc9/src/main/java/xyz/aadev/generitech/common/items/debug/ItemWorldGenWand.java#L38): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)
1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemToolWrench.java#L61](https://github.com/AlgorithmicsAnonymous/generitech/blob/7e1f33e8253730887b79a224cd65274b02577dc9/src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java#L61): Make blocksEffectiveAgainst a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemToolWrench.java#L62](https://github.com/AlgorithmicsAnonymous/generitech/blob/7e1f33e8253730887b79a224cd65274b02577dc9/src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java#L62): Make canHarvest a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemToolWrench.java#L63](https://github.com/AlgorithmicsAnonymous/generitech/blob/7e1f33e8253730887b79a224cd65274b02577dc9/src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java#L63): Make toolMaterial a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)

CrazyGrape committed:
Fixed code in compliance with SonarQube

-Changed ItemToolWrench's blocksEffectiveAgainst and toolMaterial to protected
-Refactored check on !world.isRemote to first if statement under EnumActionResult
-Removed unecessary EnumBlockRenderType import in BlockMachineMatrics.java
-Removed unused variables in ItemToolWrench

SeanLatimer commented on src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java:
```diff
@@ -58,13 +58,13 @@
-    public static ToolMaterial toolMaterial = EnumHelper.addToolMaterial("GENERITECH_WRENCH", 3, 160, 8.0F, 3.0F, 18);
-    private static IBlockState blockHarvest = null;
+    protected static Set blocksEffectiveAgainst = Sets.newHashSet(new Block[]{});
+    //public static boolean canHarvest = false;
```
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

SeanLatimer commented:
SonarQube analysis reported 3 issues
* ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) 3 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemWorldGenWand.java#L38](https://github.com/AlgorithmicsAnonymous/generitech/blob/6f65874e2b91ac25521be9cf15d1c4254339db75/src/main/java/xyz/aadev/generitech/common/items/debug/ItemWorldGenWand.java#L38): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)

SeanLatimer commented:
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemWorldGenWand.java#L38](https://github.com/AlgorithmicsAnonymous/generitech/blob/aea944800ef9741416b50b0cfd7cda79fe885da8/src/main/java/xyz/aadev/generitech/common/items/debug/ItemWorldGenWand.java#L38): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)
1. ![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) [ItemToolWrench.java#L39](https://github.com/AlgorithmicsAnonymous/generitech/blob/aea944800ef9741416b50b0cfd7cda79fe885da8/src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java#L39): Remove this unused import 'net.minecraft.block.state.IBlockState'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)

SeanLatimer committed:
Merge branch 'develop' into feature/ModelsTextures

SeanLatimer commented:
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) [ItemWorldGenWand.java#L38](https://github.com/AlgorithmicsAnonymous/generitech/blob/137d5c69dafc01695b60a704daaf3dc7123ea473/src/main/java/xyz/aadev/generitech/common/items/debug/ItemWorldGenWand.java#L38): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)
2. ![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) [ItemToolWrench.java#L39](https://github.com/AlgorithmicsAnonymous/generitech/blob/137d5c69dafc01695b60a704daaf3dc7123ea473/src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java#L39): Remove this unused import 'net.minecraft.block.state.IBlockState'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)
